src/cards_management.c

```diff
@@ -66,16 +68,24 @@ int initialize_cards(void)
     }
     g_draw_pile->next = NULL;
 
-    for (i = RED; i <= YELLOW; i++)
+	int count_wild_card = 0;
+    for (i = RED; i <= BLACK; i++)
```

Comment:
Please add action card process in another function. Don't make modification which affects the normal card.

Reply:
The action cards need to be initialized in the initialize_cards() function. The action card is defined seperately.